docs/env-providers.md

```diff
@@ -47,6 +47,20 @@ export KUBEVIRT_PROVIDER=local # choose this provider
 make cluster-up
 ```
 
+## External
+
+Uses an existing (external) Kubernetes cluster.
+
+Requires:
+ * A working Kubernetes cluster with properly configured worker nodes.
+ * A running docker daemon
+
+Usage:
+
+```bash
+export KUBEVIRT_PROVIDER=external # choose this provider
```

Comment:
If you agree with the two comments above, then configuring the provider would look like this:

```
export KUBEVIRT_PROVIDER=external
export KUBEVIRT_EXTERNAL_REGISTRY=external.registry.com
make cluster-up
make cluster-sync
```